code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -270,7 +270,7 @@
 	caliber = "a556"
 	matter = list(DEFAULT_WALL_MATERIAL = 1800)
 	ammo_type = /obj/item/ammo_casing/a556
-	max_ammo = 10
+	max_ammo = 30

```

Comment:
We already have a 20 round magazine of the 5.56 ammunition as it is. If you're going to change either of them change the bigger one.

Or make the Z8 compatible with the a556m magazines, which is probably the better route.